### PR TITLE
Replace GetSleepAnimationId with GetIdleAnimationId

### PR DESCRIPTION
**Pull request by End45**
Better documents `GetSleepAnimationId`, which is actually used to get the animation a monster should play while idling. The function has been renamed to `GetIdleAnimationId`.

Also documents `ChangeMonsterAnimation`.

### CODE REVIEW

**Review by UsernameFodder — APPROVED** [review]
Looks fine to me, will wait a day before merging since there's a symbol name being removed.

**Comment by UsernameFodder** [comment]
Oh well I guess an alias is okay too. I figured you didn't want to add one since it's not exactly correct, but I don't mind either way.

**Comment by End45** [comment]
I just added the old name as an alias (I forgot the alias system is a thing now), although I don't know if it would be better to remove it, since the old name is incorrect.

**Comment by UsernameFodder** [comment]
> I just added the old name as an alias (I forgot the alias system is a thing now), although I don't know if it would be better to remove it, since the old name is incorrect.

I kind of doubt many people use this symbol? So if I had to choose I might lean towards no alias, but will defer to you.

**Comment by End45** [comment]
Alright, I'll remove it then. I thought it was mandatory to add an alias since the checks were failing.

**Comment by UsernameFodder** [comment]
> Alright, I'll remove it then. I thought it was mandatory to add an alias since the checks were failing.

It'll only fail for the PR the change is introduced in. So it's an ignorable error.

**Pull request merged**
